Fix package name capitalisation in bitstringStatusList

The `bitstringStatusList` package currently uses a name with capital letters in the package.json:
```json
"name": "@uncefact/vckit-bitstringStatusList"
```

This causes issues with npm commands as shown by the error:
```
npm ERR! 404 This package name is not valid, because 
npm ERR! 404  1. name can no longer contain capital letters
```

## Proposed Solution
Update the package name to be all lowercase:
```json
"name": "@uncefact/vckit-bitstring-status-list"
```

--- a/packages/bitstringStatusList/README.md
+++ b/packages/bitstringStatusList/README.md
@@ -18,13 +18,13 @@ dbConnectionBitstringStatusList:
       synchronize: false
       migrationsRun: true
       migrations:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#migrations'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#migrations'
       logging: false
       entities:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#Entities'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#Entities'
 
 bitstringStatusList:
-  $require: '@uncefact/vckit-bitstringStatusList#BitstringStatusList'
+  $require: '@uncefact/vckit-bitstring-status-list#BitstringStatusList'
   $args:
     - bitstringDomainURL: http://localhost:3332
       dbConnection:
@@ -44,7 +44,7 @@ agent:
         - $require: '@veramo/credential-status#CredentialStatusPlugin'
           $args:
             - BitstringStatusListEntry:
-                $require: '@uncefact/vckit-bitstringStatusList?t=object#checkStatus'
+                $require: '@uncefact/vckit-bitstring-status-list?t=object#checkStatus'
 ```
 
 - Add the plugin to the vckit server
@@ -58,7 +58,7 @@ server:
   use:
    ...
     - - /credentials/status/bitstring-status-list
-      - $require: '@uncefact/vckit-bitstringStatusList?t=function#bitstringStatusListRouter'
+      - $require: '@uncefact/vckit-bitstring-status-list?t=function#bitstringStatusListRouter'
 ```
 
 - Add the methods to the vckit server

--- a/packages/bitstringStatusList/package.json
+++ b/packages/bitstringStatusList/package.json
@@ -1,5 +1,5 @@
 {
-  "name": "@uncefact/vckit-bitstringStatusList",
+  "name": "@uncefact/vckit-bitstring-status-list",
   "version": "1.0.0",
   "description": "Using Bitstring to represent status list for revocation VC",
   "homepage": "https://github.com/uncefact/project-vckit#readme",

--- a/packages/cli/default/agent.template.yml
+++ b/packages/cli/default/agent.template.yml
@@ -142,10 +142,10 @@ dbConnectionBitstringStatusList:
       synchronize: true
       migrationsRun: true
       migrations:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#migrations'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#migrations'
       logging: false
       entities:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#Entities'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#Entities'
 
 revocationList:
   $require: '@uncefact/vckit-revocationlist#RevocationStatus2020'
@@ -208,7 +208,7 @@ server:
       - $require: '@uncefact/vckit-revocationlist?t=function#revocationList2020Router'
 
     - - /credentials/status/bitstring-status-list
-      - $require: '@uncefact/vckit-bitstringStatusList?t=function#bitstringStatusListRouter'
+      - $require: '@uncefact/vckit-bitstring-status-list?t=function#bitstringStatusListRouter'
 
     # Agent API base path
     - - /agent
@@ -427,7 +427,7 @@ renderer:
           $require: '@uncefact/vckit-renderer#RenderTemplate2024'
 
 bitstringStatusList:
-  $require: '@uncefact/vckit-bitstringStatusList#BitstringStatusList'
+  $require: '@uncefact/vckit-bitstring-status-list#BitstringStatusList'
   $args:
     - bitstringDomainURL:
         $ref: /constants/baseUrl
@@ -465,7 +465,7 @@ agent:
         - $require: '@veramo/credential-status#CredentialStatusPlugin'
           $args:
             - BitstringStatusListEntry:
-                $require: '@uncefact/vckit-bitstringStatusList?t=object#checkStatus'
+                $require: '@uncefact/vckit-bitstring-status-list?t=object#checkStatus'
               RevocationList2020Status:
                 $require: '@uncefact/vckit-revocationlist?t=object#checkStatus'
         - $require: '@uncefact/vckit-tools#MultibaseEncodedSHA256'

--- a/packages/cli/default/default-dev.yml
+++ b/packages/cli/default/default-dev.yml
@@ -182,7 +182,7 @@ server:
       - $require: '@uncefact/vckit-revocationlist?t=function#revocationList2020Router'
 
     - - /credentials/status/bitstring-status-list
-      - $require: '@uncefact/vckit-bitstringStatusList?t=function#bitstringStatusListRouter'
+      - $require: '@uncefact/vckit-bitstring-status-list?t=function#bitstringStatusListRouter'
 
     # Agent API base path
     - - /agent
@@ -409,14 +409,14 @@ dbConnectionBitstringStatusList:
       synchronize: false
       migrationsRun: true
       migrations:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#migrations'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#migrations'
       logging: false
       entities:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#Entities'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#Entities'
 
 # BitstringStatusList for credentialStatus
 bitstringStatusList:
-  $require: '@uncefact/vckit-bitstringStatusList#BitstringStatusList'
+  $require: '@uncefact/vckit-bitstring-status-list#BitstringStatusList'
   $args:
     - bitstringDomainURL:
         $ref: /constants/baseUrl
@@ -454,7 +454,7 @@ agent:
         - $require: '@veramo/credential-status#CredentialStatusPlugin'
           $args:
             - BitstringStatusListEntry:
-                $require: '@uncefact/vckit-bitstringStatusList?t=object#checkStatus'
+                $require: '@uncefact/vckit-bitstring-status-list?t=object#checkStatus'
               RevocationList2020Status:
                 $require: '@uncefact/vckit-revocationlist?t=object#checkStatus'
         - $require: '@uncefact/vckit-tools#MultibaseEncodedSHA256'

--- a/packages/cli/default/default.yml
+++ b/packages/cli/default/default.yml
@@ -182,7 +182,7 @@ server:
       - $require: '@uncefact/vckit-revocationlist?t=function#revocationList2020Router'
 
     - - /credentials/status/bitstring-status-list
-      - $require: '@uncefact/vckit-bitstringStatusList?t=function#bitstringStatusListRouter'
+      - $require: '@uncefact/vckit-bitstring-status-list?t=function#bitstringStatusListRouter'
 
     # Agent API base path
     - - /agent
@@ -410,14 +410,14 @@ dbConnectionBitstringStatusList:
       synchronize: false
       migrationsRun: true
       migrations:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#migrations'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#migrations'
       logging: false
       entities:
-        $require: '@uncefact/vckit-bitstringStatusList?t=object#Entities'
+        $require: '@uncefact/vckit-bitstring-status-list?t=object#Entities'
 
 # BitstringStatusList for credentialStatus
 bitstringStatusList:
-  $require: '@uncefact/vckit-bitstringStatusList#BitstringStatusList'
+  $require: '@uncefact/vckit-bitstring-status-list#BitstringStatusList'
   $args:
     - bitstringDomainURL:
         $ref: /constants/baseUrl
@@ -455,7 +455,7 @@ agent:
         - $require: '@veramo/credential-status#CredentialStatusPlugin'
           $args:
             - BitstringStatusListEntry:
-                $require: '@uncefact/vckit-bitstringStatusList?t=object#checkStatus'
+                $require: '@uncefact/vckit-bitstring-status-list?t=object#checkStatus'
               RevocationList2020Status:
                 $require: '@uncefact/vckit-revocationlist?t=object#checkStatus'
         - $require: '@uncefact/vckit-tools#MultibaseEncodedSHA256'

--- a/packages/cli/package.json
+++ b/packages/cli/package.json
@@ -39,7 +39,7 @@
     "@transmute/did-context": "0.7.0-unstable.82",
     "@types/blessed": "^0.1.25",
     "@types/swagger-ui-express": "^4.1.6",
-    "@uncefact/vckit-bitstringStatusList": "workspace:^",
+    "@uncefact/vckit-bitstring-status-list": "workspace:^",
     "@uncefact/vckit-core-types": "workspace:^",
     "@uncefact/vckit-credential-data-integrity": "workspace:^",
     "@uncefact/vckit-credential-merkle-disclosure-proof": "workspace:1.0.0",

